app/models/publication.rb

```diff
@@ -203,8 +203,11 @@ def assign_uuid_and_version
     self.version ||= (publication_group.publications.maximum(:version) || 0) + 1
 
     publication_group.assign_uuid_and_number
-    publication_group.new_record? ? publication_group.latest_version = version :
-                                    publication_group.update_attribute(:latest_version, version)
+    if publication_group.new_record?
+      publication_group.latest_version = version
+    else
+      publication_group.update_column(:latest_version, version)
```

Comment:
Just confirming, we are OK skipping callbacks here?

Reply:
I hope so. These callbacks should only need to run before the first save, and they'll run when the group is saved: https://github.com/openstax/exercises/blob/main/app/models/publication_group.rb#L20-L21

`update_attribute` is supposed to skip validations, but Rails 6.1 config added something on has_many/belongs_to that validates all associated records, and that was failing here. The intention here is to just update the latest_version ASAP so other updates see the new version number.